tools/lint/lint.py

```diff
@@ -747,8 +747,18 @@ def changed_files(wpt_root):
 
 def lint_paths(kwargs, wpt_root):
     if kwargs.get("paths"):
-        r = os.path.realpath(wpt_root)
-        paths = [os.path.relpath(os.path.realpath(x), r) for x in kwargs["paths"]]
+        #make new list in case there are multiple added directories
+        paths = []
+        for path in kwargs.get("paths"):
+            if os.path.isdir(path):
+            # create extension for called directory and merge it with the existing path
+                new_path = os.path.realpath(os.path.join(wpt_root, path))
```

Comment:
So this assumes that `path` is relative to the `repo_root` but above it's assumed to be relative to the current working directory. I think the latter is correct.

Reply:
The later is correct. It should have been correct here as well, new_path should be the former.